webpack-blocks.js

```diff
@@ -0,0 +1,88 @@
+const { entryPoint, group, setOutput, sourceMaps } = require('@webpack-blocks/webpack2');
```

Comment:
More specifically I'm wondering if you have any better ideas for this file. It's shared code for the webpack configs.